design/one-pager-package-runtime-config.md

```diff
@@ -0,0 +1,222 @@
+# Package Runtime Config
+
+* Owner: Nic Cope (@negz)
+* Reviewers: Hasan Turken (@turkenh), Dan Mangum (@hasheddan)
+* Status: Accepted
+
+## Background
+
+Crossplane Providers are Kubernetes controller managers. They're packaged in an
+OCI container and expect to connect to an API server. You install a Provider
+declaratively by creating a Provider resource. When you do, the Crossplane
+package manager installs your provider by creating a Kubernetes Deployment,
+along with some accoutrements like a ServiceAccount and a Service. The RBAC
+manager also creates some RBAC ClusterRoles and ClusterRoleBindings to allow the
+Provider's ServiceAccount (and thus Deployment) to do what it needs to do.
+
+There are two things we'd like to improve about the way this works today:
+
+1. Folks want more control over how their Providers are deployed - over the
+   configuration of the Deployment (etc) Crossplane creates. ([#3601])
+2. In some cases, folks don't want Crossplane to create a Kubernetes Deployment
+   at all. They want to run a Provider some other way. ([#2671])
+
+Soon [Composition Functions][functions-beta-design] will also be deployed by the
+package manager as Kubernetes Deployments. We expect they'll have similar
+requirements. Since Functions aren't Kubernetes controllers, in this document
+I'll refer to the long-running processes some packages need to deploy as
+'package runtimes'.
```

Comment:
nitpick: I would think of the component that deploys and manages the package processes (or configuration of them) as the package runtime. In the default case in this proposal, Kubernetes (Deployments) would be the package runtime rather than each individual Deployment that gets created (e.g. `containerd` is a runtime for containerized processes, the containerized processes themselves are not a runtime)